public/doc/lishogi-api.yaml

```diff
@@ -4405,2267 +3153,236 @@ paths:
-        "200":
-          description: The draw offer was successfully sent.
-          headers:
-            Access-Control-Allow-Origin:
-              schema:
-                type: string
-                default: "'*'"
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Ok'
-        "400":
-          description: The draw offering failed.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-
-  /api/board/game/{gameId}/takeback/{accept}:
-    post:
-      operationId: boardGameTakeback
-      summary: Handle takeback offers
-      description: |
-        Create/accept/decline takebacks.
-        - `yes`: Propose a takeback, or accept the opponent's takeback offer.
-        - `no`: Decline a takeback offer from the opponent.
-      tags:
-        - Board
-      security:
-        - OAuth2: ["board:play"]
-      parameters:
-        - in: path
-          name: gameId
-          schema:
-            type: string
-            example: "5IrD6Gzz"
-          required: true
-        - in: path
-          name: accept
-          schema:
-            anyOf:
-              - type: boolean
-              - type: string
-                const: yes
-            example: "yes"
-          required: true
-      responses:
-        "200":
-          description: The takeback offer was successfully sent.
-          headers:
-            Access-Control-Allow-Origin:
-              schema:
-                type: string
-                default: "'*'"
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Ok'
-        "400":
-          description: The takeback offering failed.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-
-  /api/board/game/{gameId}/claim-victory:
-    post:
-      operationId: boardGameClaimVictory
-      summary: Claim victory of a game
-      description: |
-        Claim victory when the opponent has left the game for a while.
-      tags:
-        - Board
-      security:
-        - OAuth2: ["board:play"]
-      parameters:
-        - in: path
-          name: gameId
-          schema:
-            type: string
-            example: "5IrD6Gzz"
-          required: true
-      responses:
-        "200":
-          description: The victory was successfully claimed.
-          headers:
-            Access-Control-Allow-Origin:
-              schema:
-                type: string
-                default: "'*'"
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Ok'
-        "400":
-          description: The victory claim has failed.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-
-  /api/bot/online:
```

Comment:
Please see #580

Reply:
https://github.com/lichess-org/lila/commit/4df218c7ede4431cf30f9b8c7b78c1cb391cc1c5 Isn't this the lichess commit? Does PR add the endpoint `/api/bot/online` or it directs to some other endpoint?

Reply:
Yes, this PR adds the GET /api/bot endpoint:
```
GET   /api/bot/*cmd                    controllers.PlayApi.botCommandGet(cmd: String)
```

Reply:
https://github.com/TheYoBots/lishogi/blob/lishogi-api-doc/public/doc/lishogi-api.yaml#L2790 I hadn't removed the endpoint that you added in your PR.

Reply:
My PR adds all `GET /api/bot/*` paths, I think; previously there were none.

Reply:
As per what I see in your PR here https://github.com/WandererXII/lishogi/pull/580/files all changes has something to do with chat. The name of the commit too is to add the endpoint `GET /api/bot/game/{gameId}/chat`.

Incase you want to add the endpoint `GET api/bot/online` then the lichess commit you are looking for is this https://github.com/lichess-org/lila/commit/4df218c7ede4431cf30f9b8c7b78c1cb391cc1c5

Reply:
Thanks, you are correct.  I've fixed my PR and my test site http://dugovic.mooo.com/ is running, although I do not know how to test a Lishogi BOT (can connect and play on it).